### PR TITLE
Revert "Revert "Created scheduled classes table""

### PR DESCRIPTION
**Pull request by aagrawal05**
@Hydrohaven @h0ethan04

389251694-14b14cf7-15aa-4a83-9239-2cc628045862

Our actual PR now is to revert your reversion so that our original changes are reflected
[our current branch is now technically behind dev with no new commits that can be merged]

### CODE REVIEW

File: server/db/schema/scheduled_classes.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS scheduled_classes CASCADE;
+
+CREATE TABLE scheduled_classes (
+  class_id INTEGER NOT NULL,
+  date DATE NOT NULL,
+  start_time TIME NOT NULL,
+  end_time TIME NOT NULL,
+  PRIMARY KEY (class_id, date),
+  FOREIGN KEY (class_id) REFERENCES classes(id)
```
Comment (h0ethan04) [inline server/db/schema/scheduled_classes.sql]: Whenever we use foreign keys we should be cascading changes on delete, can you add that?

**Review by h0ethan04 — CHANGES_REQUESTED** [review]

File: server/db/schema/scheduled_classes.sql (hunk 1 above)
Comment (aagrawal05) [reply server/db/schema/scheduled_classes.sql]: gotcha

**Comment by aagrawal05** [comment]
cascading delete on foreign key should be completed now

**Review by h0ethan04 — APPROVED** [review]
lgtm!

**Pull request merged**